components/MemberCard.tsx

```diff
@@ -0,0 +1,36 @@
+import { StaticImageData } from "next/image";
+import Image from "next/image";
+import { useState } from "react";
+
+const MemberCard = ({imageSrc, firstName, lastName, description, position}: MemberCardProps) => {
+    const [hovered, setHovered] = useState(false);
+
+    return(
+        <div className="flex flex-col items-center justify-center rounded bg-wato-grey-porcelain lg:w-[8vw] lg:min-w-[8vw]"
```

Comment:
Let's just handle width using the grid. This way the gap between elements will be equal vertically and horizontally.
Let's also add some padding
```jsx
<div className="flex flex-col items-center justify-center rounded bg-wato-grey-porcelain py-3">
```